src/io_hid_keys.rs

```diff
@@ -0,0 +1,9 @@
+// exports from <IOKit/hid/IOHIDKeys.h>
+
+use libc::c_int;
+
+pub type IOHIDReportType = c_int;
+pub const kIOHIDReportTypeInput: IOHIDReportType   = 0x0;
+pub const kIOHIDReportTypeOutput: IOHIDReportType  = 0x1;
+pub const kIOHIDReportTypeFeature: IOHIDReportType = 0x2;
+pub const kIOHIDReportTypeCount: IOHIDReportType   = 0x3;
```

Comment:
This enum isn't a bitflag. It's defined in the header like this:

```c
enum IOHIDReportType{
    kIOHIDReportTypeInput = 0,
    kIOHIDReportTypeOutput,
    kIOHIDReportTypeFeature,
    kIOHIDReportTypeCount
};
```

My prior comment was targetted at `IOHIDManagerOptions` only.